spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/AboutController.java

```diff
@@ -15,28 +15,20 @@
  */
 package org.springframework.cloud.dataflow.server.controller;
 
-import java.util.ArrayList;
-import java.util.List;
-
 import org.apache.http.conn.ssl.NoopHostnameVerifier;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClients;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
+import org.springframework.beans.factory.ObjectProvider;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.info.BuildInfoContributor;
+import org.springframework.boot.actuate.info.GitInfoContributor;
+import org.springframework.boot.actuate.info.Info;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.cloud.common.security.support.SecurityStateBean;
 import org.springframework.cloud.dataflow.core.Launcher;
-import org.springframework.cloud.dataflow.rest.resource.about.AboutResource;
-import org.springframework.cloud.dataflow.rest.resource.about.Dependency;
-import org.springframework.cloud.dataflow.rest.resource.about.FeatureInfo;
-import org.springframework.cloud.dataflow.rest.resource.about.MonitoringDashboardInfo;
-import org.springframework.cloud.dataflow.rest.resource.about.MonitoringDashboardType;
-import org.springframework.cloud.dataflow.rest.resource.about.RuntimeEnvironment;
-import org.springframework.cloud.dataflow.rest.resource.about.RuntimeEnvironmentDetails;
-import org.springframework.cloud.dataflow.rest.resource.about.SecurityInfo;
-import org.springframework.cloud.dataflow.rest.resource.about.VersionInfo;
+import org.springframework.cloud.dataflow.rest.resource.about.*;
```

Comment:
Please replace this w/ the specific imports. There is a setting in IDEA where you can disable this wildard imports.

Reply:
Yes... I forgot about that, I've just changed that option... soon a new commit.